src/index.ts

```diff
@@ -34,11 +34,12 @@ import {typeOf} from './expression/values';
 import {FormatExpression} from './expression/definitions/format';
 import {Literal} from './expression/definitions/literal';
 import {CompoundExpression} from './expression/compound_expression';
-import {VariableAnchorOffsetCollectionSpecification} from './types.g';
+import {ProjectionDefinitionSpecification, VariableAnchorOffsetCollectionSpecification} from './types.g';
```

Comment:
I would expect to see here more specifications imports like padding? color?
I think the `StylePropertySpecification` can be improved a bit to use those...?

Reply:
I've fixes that